pydrad/parse/parse.py

```diff
@@ -46,7 +62,8 @@ def __init__(self, hydrad_root, **kwargs):
         # when slicing.
         self._master_time = kwargs.pop('master_time', None)
         if self._master_time is None:
-            self._master_time = get_master_time(self.hydrad_root)
+            self._master_time = get_master_time(self.hydrad_root,
+                                                read_from_cfg=kwargs.get('read_from_cfg'))
```

Comment:
```suggestion
                                                read_from_cfg=kwargs.get('read_from_cfg', False))
```

Reply:
Does this just set the default to False?

Reply:
Yes. If you don't put anything there, it will return `None`